CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing
+
+## Getting Started
+
+General information about setting up your Python environment, testing modules,
+Ansible coding styles, and more can be found in the [Ansible Community Guide](
+https://docs.ansible.com/ansible/latest/community/index.html).
+
+
+## ServiceNow ITSM Collections
+
+### servicenow.itsm
+
+The Ansible Collection for ServiceNow IT Service Management ([ITSM](https://www.servicenow.com/products/itsm.html)) includes a variety of Ansible content to help automate the management of ServiceNow IT Service Management.
+
+New modules and plugins developed by the community should be proposed to `servicenow.itsm`.
+
+## Submitting Issues
+All software has bugs, and the `servicenow.itsm` collection is no exception. When you find a bug,
+you can help tremendously by [telling us about it](https://github.com/ansible-collections/servicenow.itsm/issues).
+
+If you should discover that the bug you're trying to file already exists in an issue,
+you can help by verifying the behavior of the reported bug with a comment in that
+issue, or by reporting any additional information.
+
+## Writing New Code
+
+New modules should be submitted to the [servicenow.itsm](https://github.com/ansible-collections/servicenow.itsm) collection.
+
+### Static analysis
```

Comment:
This section title doesn't seem right to me. I thought this doc didn't address new features and bug fixes until I looked again. kubernetes.core just has a ["Pull Requests" section](https://github.com/ansible-collections/kubernetes.core/blob/main/CONTRIBUTING.md#pull-requests) the encompasses new code and features/bugs. amazon.aws has it all under one ["Writing new code" section](https://github.com/ansible-collections/kubernetes.core/blob/main/CONTRIBUTING.md#pull-requests).